build/musicbrainz-dev/scripts/update-javascript.sh

```diff
@@ -8,3 +8,4 @@ yarn
 
 dockerize -wait tcp://db:5432 -timeout 60s -wait tcp://mq:5672 -timeout 60s -wait tcp://redis:6379 -timeout 60s ./script/compile_resources.sh
 
+./script/compile_resources.sh --watch &> /compile_resources.log &
```

Comment:
Note that this also triggers a new build from scratch, so it's partially redundant with the ./script/compile_resources.sh above (it'll build in full twice, then watch the second build).  Maybe change the final dockerize argument to a no-op command like `true` or something?

Reply:
Thanks, I noticed it was redundant but I noticed issues when running in watch mode directly (which was probably why I used to run watch mode in a separate shell for so long). While reading your comment I just realized that these issues were solely due to running `dockerize` in the background rather than waiting for it before moving on with starting the server! No need of `true` though, the command argument to `dockerize` is just optional.

Fixed in <https://github.com/metabrainz/musicbrainz-docker/pull/205/commits/9f4bf8509fe0cb758551bd221a09be36558d3e84>.